llvm/utils/git/code-format-helper.py

```diff
@@ -63,17 +99,16 @@ def pr_comment_text_for_diff(self, diff: str) -> str:
 </details>
 """
 
-    def find_comment(
-        self, pr: PullRequest.PullRequest
-    ) -> IssueComment.IssueComment | None:
+    def find_comment(self, pr: any) -> any:
```

Comment:
It would be nice if we could maintain the type annotations here even though we don't want to import `github` when running as a hook. Probably would require some refactoring that can be left for future work though. Maybe leave a todo?